geodiff/src/drivers/sqlitedriver.cpp

```diff
@@ -417,8 +417,17 @@ static std::string sqlFindModified( const std::string &tableName, const TableSch
     {
       if ( !exprOther.empty() )
         exprOther += " OR ";
-      exprOther += sqlitePrintf( "\"%w\".\"%w\".\"%w\" IS NOT \"%w\".\"%w\".\"%w\"",
-                                 "main", tableName.c_str(), c.name.c_str(), "aux", tableName.c_str(), c.name.c_str() );
+
+      if ( c.type.dbType == "DATETIME" )
+      {
+        exprOther += sqlitePrintf( "STRFTIME('%%Y-%%m-%%d %%H:%%M:%%S.%%fZ', \"%w\".\"%w\".\"%w\") != STRFTIME('%%Y-%%m-%%d %%H:%%M:%%S.%%fZ', \"%w\".\"%w\".\"%w\")",
+                                   "main", tableName.c_str(), c.name.c_str(), "aux", tableName.c_str(), c.name.c_str() );
```

Comment:
Is it really necessary to have this strftime-based check, and not just the generic `a IS NOT b` below? (which tests exact match, and there's still secondary check in handleUpdated()...

Reply:
I just wanted to be safe here. Removed.